ui/raidboss/data/07-dt/trial/queen-eternal-ex.txt

```diff
@@ -0,0 +1,211 @@
+# -p A04B:1000.0 -ii A008 A009 A00D A00E A00F A010 A018 A026 A027 A02F A030 A053 A054 A056
+
+# Phase one pulled from -r XwHyk2NAKnbzZLf7 -rf 3
+# Phase two pulled from -r GBaArv7yzHnXck8D -rf 17
+# A lot of manual cleanup applied
+
+# Phase one
+0.0 "--sync--" InCombat { inGameCombat: "1" } window 0,1
+12.9 "--sync--" StartsUsing { id: "A00C", source: "Queen Eternal" }
+15.6 "Aethertithe" Ability { id: "A00C", source: "Queen Eternal" } duration 27.8
+27.4 "Retribute" Ability { id: "A011", source: "Queen Eternal" }
+28.3 "Retribute" #Ability { id: "A012", source: "Queen Eternal" }
+36.4 "Retribute" Ability { id: "A011", source: "Queen Eternal" }
+37.3 "Retribute" #Ability { id: "A012", source: "Queen Eternal" }
+45.4 "Retribute" Ability { id: "A011", source: "Queen Eternal" }
+46.3 "Retribute" #Ability { id: "A012", source: "Queen Eternal" }
```

Comment:
I haven't had time to run this yet, but these are the light party stacks, correct? If so, it might make more sense to `--sync--` the non-damage instances and then label the others as 1-2-3.

Reply:
Here and elsewhere with the double casts, I'm just dropping the one that doesn't really have any effect, as they're just clutter in the timeline with no benefit.

In this case it's the `A011` ability. There's no cast bar at all for these light parties, so there's not even a case of `(cast)` vs `(snapshot)` or whatever, there's just an extra `A011` for no apparent reason for these.